### PR TITLE
numa_opts: Update default memory size for 3 nodes test

### PR DESCRIPTION
**Pull request by huangyum**
ID: 1500635
Signed-off-by: Yumei Huang <yuhuang@redhat.com>

### CODE REVIEW

**Comment by jingzhao84** [comment]
@huangyum could you paste the test result ?

File: qemu/tests/cfg/numa_opts.cfg [hunk 1]
```diff
@@ -81,8 +81,8 @@
           numa_node1_cpus = "3 4 5"
           numa_node2_cpus = ""
           numa_node0_size = 80
-          numa_node1_size = 80
-          numa_node2_size = 96
+          numa_node1_size = 88
+          numa_node2_size = 88
```
Comment (hereischen) [inline qemu/tests/cfg/numa_opts.cfg]: it is 98 in BZ

**Comment by huangyum** [comment]
(1/2) smp_16.3128m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.numa_opts.nodes.3.defaults: PASS (8.31 s)
 (2/2) smp_16.3128m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.Win2016.x86_64.io-github-autotest-qemu.numa_opts.nodes.3.defaults: PASS (7.28 s)

**Comment by jingzhao84** [comment]
Acked-by: jinzhao (jinzhao@redhat.com)

**Comment by suqinhuang** [comment]
ack

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**